zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowModel.java

```diff
@@ -113,7 +113,12 @@ public static TensorFlowModel create(final Model.Id id,
     final URI localDir = FileSystemExtras.downloadIfNonLocal(normalizedUri);
     final SavedModelBundle model = SavedModelBundle.load(localDir.toString(),
                                                          options.tags().toArray(new String[0]));
-    final MetaGraphDef metaGraphDef = extractMetaGraphDefinition(model);
+    final MetaGraphDef metaGraphDef;
+    try {
+      metaGraphDef = extractMetaGraphDefinition(model);
+    } catch (TensorflowMetagraphParsingException e) {
+      throw new RuntimeException(e);
```

Comment:
since this effectively makes the loading of a model fail we could throw a new IOException here with TensorflowMetagraphParsingException as root cause.

Reply:
Good point